node/_fs/_fs_access.ts

```diff
@@ -1,7 +1,8 @@
 // Copyright 2018-2022 the Deno authors. All rights reserved. MIT license.
 
 import { type CallbackWithError, makeCallback } from "./_fs_common.ts";
-import { fs, os } from "../internal_binding/constants.ts";
```

Comment:
All this instances of `os.errno` were incorrect. In Node the error code is based on the codes from "uv" rather than "constants"